src/django/pfb_analysis/management/commands/generate_analysis_csv.py

```diff
@@ -0,0 +1,124 @@
+    return (columns, values,)
+
+
+def export_mileage_low_stress(job):
+    value = job.overall_scores['total_miles_low_stress']['score_normalized']
+    return (['total_low_stress_miles'], [value],)
+
+
+def export_mileage_high_stress(job):
+    value = job.overall_scores['total_miles_high_stress']['score_normalized']
+    return (['total_high_stress_miles'], [value],)
+
+
+def export_total_population(job):
+    return (['total_population'], [job.population_total],)
+
+
+def export_boundary_area(job):
+    MILES_PER_METER = 0.000621371
+    return (['neighborhood_area_m2'],
+            [job.neighborhood.geom_utm.area * MILES_PER_METER * MILES_PER_METER],)
+
+
+# This should be an array of functions that take an AnalysisJob instance as their only argument
+#  and return a tuple, where the first entry is an array of column names to add to the csv,
+#  and the second entry is the associated values for each column in the first tuple entry.
+EXPORTS = [
+    export_job_details,
+    export_connectivity_metrics,
+    export_mileage_high_stress,
+    export_mileage_low_stress,
+    export_total_population,
+    export_boundary_area,
+]
+
+
+class Command(BaseCommand):
+    help = """ Generate a CSV summary of AnalysisJob results and push to AWS S3
+
+    Generates a row in the CSV for each Neighborhood whose last job has status=COMPLETE
+
+    CSV contains the following metrics:
+    - Overall score
+    - Each connectivity metric
+    - Mileage of low-stress segments
+    - Mileage of high-stress segments
+    - Neighborhood total population
+    - Neighborhood boundary land area in square meters
+
+    """
+
+    def add_arguments(self, parser):
+        pass
+
+    def handle(self, *args, **options):
+
+        tmpdir = tempfile.mkdtemp()
+
+        queryset = AnalysisJob.objects.all().filter(status=AnalysisJob.Status.COMPLETE)
+        filter_set = AnalysisJobFilterSet()
+        queryset = filter_set.filter_latest(queryset, 'latest', True)
+
+        tmp_csv_filename = os.path.join(tmpdir, 'results.csv')
+        with open(tmp_csv_filename, 'w') as csv_file:
+            writer = None
+            fieldnames = []
+
+            for job in queryset:
+                row_data = {}
+                for export in EXPORTS:
+                    columns, values = export(job)
+                    if writer is None:
+                        fieldnames = fieldnames + columns
+                    for column, value in zip(columns, values):
+                        row_data[column] = value
+                if writer is None:
+                    logger.debug(fieldnames)
+                    writer = csv.DictWriter(csv_file,
+                                            fieldnames=fieldnames,
+                                            dialect=csv.excel,
+                                            quoting=csv.QUOTE_MINIMAL)
+                    writer.writeheader()
+                logger.debug(row_data)
```

Comment:
I had left them in intentionally, but there's probably not a lot of value in doing so. Will remove.